vapi/config.vapi

```diff
@@ -1,7 +1,5 @@
 [CCode (cprefix = "", lower_case_cprefix = "", cheader_filename = "config.h")]
 namespace Build {
-	public const string DATADIR;
-	public const string PKGDATADIR;
-	public const string GETTEXT_PACKAGE;
-	public const string VERSION;
```

Comment:
To my knowledge, only the `PKGDATADIR` is called in the code. Can you tell me why we would keep `GETTEXT_PACKAGE` as well?

I would like to keep `VERSION` for a future CLI flag feature.

Reply:
I've once removed `GETTEXT_PACKAGE` in my previous "Add Meson" PR, but the CI complains its unexistence, although I've removed codes that uses it: https://travis-ci.org/dahenson/agenda/builds/612755020#L3463-L3469.

However, I'll remove this const again and see what happens :smile: